admin/client/src/state/formData/README.md

```diff
@@ -0,0 +1,9 @@
+# Form Data state
+
+Uses `formData` store.
+
+Manages state associated with the FormBuilderLoader.
+
+This sets form data that does not quite belong in schema, as it is client-side generated data, as opposed to server-side generated for schema.
```

Comment:
Calling this `formData` is quite misleading, given that the *actual* form data (form field values) is located in `schema`. How about a state structure like `schema: { schema: {}, state: {}, metaData: {} }`? It's hard to name this, since it contains both metadata (`loading`) and data (`stateOverride`).